app/src/main/java/io/pslab/activity/MultimeterActivity.java

```diff
@@ -110,8 +116,17 @@
     private boolean locationEnabled = true;
     private long recordPeriod;
     private double lat = 0, lon = 0;
-    private String multimeterCSVheader = "Data,Value,Latitude,Longitude\n";
+    private String multimeterCSVheader = "Timestamp,DateTime,Data,Value,Latitude,Longitude";
```

Comment:
Why are we adding a timestamp and a datetime?

Reply:
To have consistent xsv format as other instruments